posts/2024-01-31-Performance-Doris_Amoakohene/index.qmd

```diff
@@ -26,12 +27,12 @@ In data.table, the term “Performance Regression” usually means a drop in bot
 
 # **Why do we run Performance Tests on commits?**
 
-Running performance tests on commits helps maintain a high-performance standard for the package, detect and fix performance regressions, optimize code, validate performance improvements,ensure consistent performance over time and to encourage confidence in code contributions from new people.
+Running performance tests on commits helps maintain a high-performance standard for the package, detect and fix performance regressions, optimize code, validate performance improvements, ensure consistent performance over time and to encourage confidence in code contributions from new people.
 
 It is an essential practice to deliver a performant and reliable package to end-users.
 
 # **What are the Performance Tests?**
-Our atime Performance Tests aim to assess the efficiency of the data.table Package by simulating the data.table repository and gathering information on memory and time usage.It also  evaluate how effectively the package performs under various scenarios and workloads. 
+Our atime Performance Tests aim to assess the efficiency of the data.table Package by simulating the data.table repository and gathering information on memory and time usage. It also evaluate how effectively the package performs under various scenarios and workloads. 
```

Comment:
please clarify "simulating" what does that mean?

Reply:
updated